pkgs/by-name/lo/lora/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation rec {
```

```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```
- #315337
- https://nix.dev/guides/best-practices#recursive-attribute-set-rec